tests/behat/features/bootstrap/ApiNewFeatureContext.php

```diff
@@ -0,0 +1,301 @@
+<?php
```

Comment:
Many `use` statements that are not really used.
Pretty sure there is a cleaner way to write those context files. But the context file clean up process is not important. You can leave it as it is for now.